packages/dx-grid-core/src/plugins/table-column-resizing/reducers.ts

```diff
@@ -1,28 +1,52 @@
 import { slice } from '@devexpress/dx-core';
-import { ColumnWidthReducer } from '../../types';
+import { ColumnWidthReducer, ColumnSizeFn } from '../../types';
+
+export const getColumnSize: ColumnSizeFn = (
+  updatedColumn, {columnName, shift, minColumnWidth, maxColumnWidth, columnExtensions,
+}) => {
+  const indexOfExtended = columnExtensions
```

Comment:
Specify the `columnExtensions` default value, an empty array. Then ternary operator can be removed

Reply:
And we can use `find` here